plugin/adsk/scripts/USDMenuProc.mel

```diff
@@ -76,6 +76,14 @@ global proc USDMenuProc(string $parent, string $obj)
         }
         menuItem -label "Duplicate As Maya Data" -command ("{waitCursor -state 1; mayaUsdMenu_duplicate \"" + $obj + "\"; waitCursor -state 0;}");
     }
+
+    // Allow LookdevX to define a proc with additional menu items
```

Comment:
Not clean. Requires secret functions shared with a third party.
Please create a publicly known function to register additional menu procs, then use the list of registered procs here. The third party will then be able to register a proc without MayaUSD having to know about it.

Reply:
This was code added to follow an existing style already used in Maya to call a user defined menu proc. What Jerry is suggesting is better for what you are trying to add which is a 3rd party (not user).

Reply:
Actually `USDUserMenuProc` was added specifically for the needs of LookdevX (and also other third parties, as seen in #1773), so my suggestion is that LookdevX should use `USDUserMenuProc` until we get news about another plugin hijacking it.

MAYA-125745 should be about finding a way to make that extensible enough that two plugins can register their menu hooks simultaneously.